src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -8,6 +8,7 @@ $(function () {
   var hide = 'hide';
   var util = new utility();
 
+  workAroundFixedHeaderForAnchors();
```

Comment:
naming: lowercase `Around`'s `a` as `workaround` is a word

Reply:
"workaround" is a noun but when it's a verb it's "work around".